go.sum

```diff
@@ -457,7 +449,7 @@ golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2/go.mod h1:NqM8EUOU14njkJ3
 golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk=
 golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
 golang.org/x/text v0.3.6/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
-golang.org/x/text v0.3.8/go.mod h1:E6s5w1FMmriuDzIBO73fBruAKo1PCIq6d2Q6DHfQ8WQ=
+golang.org/x/text v0.3.7/go.mod h1:u+2+/6zg+i71rQMx5EYifcz6MCKuco9NR6JIITiCfzQ=
```

Comment:
is 0.3.7 ok to use? https://pkg.go.dev/vuln/GO-2022-1059

Reply:
I think it's fine, it's used indirectly by some other indirect dependency and the scanners are not detecting it.

Reply:
I see. but which one is going to be used when building the binary?

Reply:
As I understand it to build our binary we would use `golang.org/x/text v0.6.0` since it's in our indirect dependencies. 
The `go.sum` file just contains all of the hashes of all the versions used by our dependencies and their dependencies and their dependencies etc all the way to the end 😅

Reply:
@lucacome thanks for explaining! 👍  I got a bit confused

Reply:
of course! 😄